### DISCUSSION TITLE
help with y-overflow and the max-height of an element

### DISCUSSION THREAD

**Discussion by fo-od** [tailwindlabs/tailwindcss#12843] [Help] [answered]
So im trying to make the element have a y overflow, and I was setting the max height to be full, but it just makes the page longer instead of letting you scroll on the element. the element I am trying to do this to is inside of a flexbox that has min-h-screen.
tailwindplay link: https://play.tailwindcss.com/g0ofH8dSSM

**Comment by wongjn** [comment, marked as answer]
`min-h-screen` means `min-height: 100vh` which means be *at least* screen height, but let the element be longer if it so wishes. You have the element limited to a *maximum* of `100vh`, consider applying `height` or `max-height` instead of `min-height`. See [this modified version of your Tailwind Play](https://play.tailwindcss.com/EbjKeo32xL).

**Reply by fo-od** [reply]
thanks !!!

**Reply by fo-od** [reply]
it seems to have an issue though, 
![image](https://github.com/tailwindlabs/tailwindcss/assets/99752044/54c99de9-d194-423c-a26b-1031b1cbaaaa)

**Reply by fo-od** [reply]
(it goes over the header)

**Reply by wongjn** [reply]
Yes, it would do that with how you have your layout.

**Reply by fo-od** [reply]
is there a way to fix it?

**Reply by wongjn** [reply]
You could go back to `min-h-screen`, but then you will lose the scrolling.

**Reply by fo-od** [reply]
oh dang, i guess ill just deal with it then
